tests/dhcp-eve-extended-option-60/test.yaml

```diff
@@ -0,0 +1,41 @@
+requires:
+  min-version: 7
+
+args:
+- -k none --set outputs.1.eve-log.types.21.dhcp.extended=yes
```

Comment:
@jasonish will it always be `21` ?

Or should rather a minimal suricata.yaml be used here ?

Reply:
I'd rather a minimal YAML.  We do have existing tests digging into arrays like this, but it is fragile.  @yatink Could you update this test to provide a minimal YAML file instead of the `--set`?

Reply:
Sure, I can read the docs and figure out how to add that. I'll ask here if I have questions.

Reply:
Look out for other Suricata Verify tests having a suricata.yaml in their directory

Reply:
One such possible example: https://github.com/OISF/suricata-verify/blob/master/tests/bug-5802/suricata.yaml :)

Reply:
Ok done....I'll squash my changes and open a new PR.